3.2.js

```diff
@@ -0,0 +1,14 @@
+function isEven(num) {
+    if (num == 0) {
+        return true;
+    }
+    else if (num == 1) {

```

Comment:
po co Ci "else" jesli wyzej w ifie masz return?

btw. else powinno byc w tej samej linii co zamkniecie klamry od ifa

to do "else" i do "==" dotyczy wszystkich zadan :)